roles/openshift_aws/tasks/provision_instance.yml

```diff
@@ -1,4 +1,20 @@
 ---
+- name: set openshift_node_bootstrap to True when building AMI
+  set_fact:
+    openshift_node_bootstrap: True
+
+- when: openshift_aws_create_vpc | bool
```

Comment:
@michaelgugino, this PR redates the PRs that rearranged the code.  I can remove the VPC pieces as they now happen in the playbook instead of the role.